tests/storages_tests/test_storages.py

```diff
@@ -158,7 +158,6 @@ def test_get_study_id_from_name_and_get_study_name_from_id(storage_mode: str) ->
         # Generate unique study_name from the current function name and storage_mode.
         function_name = test_get_study_id_from_name_and_get_study_name_from_id.__name__
         study_name = function_name + "/" + storage_mode
-        storage = optuna.storages.get_storage(storage)
```

Comment:
Why this line is deleted? Many other calls to get_storage have also been removed, why?

Reply:
英語で正確に説明する自信が無かったので日本語にて失礼します。

この PR は ```RedisStorage``` を ```CachedStorage``` で wrap しており、この修正は test coverage を悪化させます。理由は、いくつかの関数（今まで直接 ```RedisStorage``` を使用していた場合は呼ばれていた関数）が呼ばれなくなるからです。

```RDBStorage``` についても同様に、```CachedStorage``` で wrap した際に、同じ関数が呼ばれなくなります。

最初の commit においては、 test coverage が悪化することは認識していたものの、 ```RDBStorage``` と同じレベルに悪化するので問題ないだろうと思い、そのままの状態で PR を送りました。

しかし、 test coverage を改善するようにとのご指摘があり、改善のために、 ```RDBStorage/RedisStorage``` の呼ばれない関数を obsolete にしてみました。

しかし、 ```RDBStorage/RedisStorage``` を ```CachedStorage``` で wrap せずに使う可能性があるので、それはやめて欲しいとのご指摘がありました。

そこで、 test coverage の改善のために、 ```RDBStorage/RedisStorage``` を直接使用するテストケースを追加しました。いくつかのテストにおいて ```get_storage()``` を削除しているのはそのためです。```get_storage()``` は ```CachedStorage``` で ```RDBStorage/RedisStorage``` を wrap してしまうので、それを避ける必要がありました。

Reply:
This PR wraps `RedisStorage` with `CachedStorage`, and this PR worsens the test coverage. The reason is that some functions (that would have been called if you were using `RedisStorage` directly) will no longer be called.

Similarly for `RDBStorage`, the same functions will no longer be called when wrapping with `CachedStorage`.

In the first commit, I was aware that the test coverage would deteriorate, but I thought it would be fine since it would deteriorate to the same level as `RDBStorage`, so I sent the PR as is.

However, I was asked to improve the test coverage, so I tried to obsolete the functions that are not called in `RDBStorage/RedisStorage` in order to improve it.

However, it was pointed out that `RDBStorage/RedisStorage` may be used without wrapping in `CachedStorage`, and they don't want that to happen.

So, to improve the test coverage, I have added test cases that use `RDBStorage/RedisStorage` directly. That's why I removed `get_storage()` in some tests, because `get_storage()` wraps `RDBStorage/RedisStorage` in `CachedStorage`, and we needed to avoid that.

Reply:
Thank you for the explanation. I agree with you.